server/kitsu/anatomy.py

```diff
@@ -212,13 +216,18 @@ async def get_kitsu_project_anatomy(
     statuses = await parse_statuses(addon, kitsu_project_id)
     task_types = await parse_task_types(addon, kitsu_project_id)
 
-    anatomy_preset = await get_primary_anatomy_preset()
-    anatomy_dict = anatomy_preset.dict()
+    if ayon_project:
+        anatomy = extract_ayon_project_anatomy(ayon_project)
+    else:
+        anatomy = await get_primary_anatomy_preset()
+
+    anatomy_dict = anatomy.dict()
     for key in anatomy_dict["attributes"]:
         if key in attributes:
-            anatomy_dict["attributes"][key]=attributes[key]
+            anatomy_dict["attributes"][key] = attributes[key]
+            logging.debug("updated", key, "to", attributes[key])
```

Comment:
Should this report the project name and maybe mention that it's updating anatomy?
Psuedocode suggestion:
```suggestion
            logging.debug("updated project", ayon_project.name, "anatomy attribute", key, "to", attributes[key])
```